docs/src/pages/components/data-grid/group-pivot/group-pivot.md

```diff
@@ -53,13 +238,11 @@ const rows: GridRowsProp = [
 />;
 ```
 
-{{"demo": "pages/components/data-grid/group-pivot/BasicTreeData.js", "bg": "inline", "defaultCodeOpen": false}}
+{{"demo": "pages/components/data-grid/group-pivot/TreeDataSimple.js", "bg": "inline", "defaultCodeOpen": false}}
 
 ### Custom grouping column
 
-Use the `groupingColDef` prop to customize the rendering of the grouping column.
-
-{{"demo": "pages/components/data-grid/group-pivot/CustomGroupingColumnTreeData.js", "bg": "inline", "defaultCodeOpen": false}}
+Same behavior as for the [Row grouping](##grouping-column-customization) except for the `leafField` and `mainGroupingCriteria` which are not applicable for the Tree Data.
```

Comment:
This link is pointing to himself. The title you want to refer to has the same name but is `h4`. So it does not have ancor link.

Reply:
I'm not sure it is a good idea to replace [the tree demo](https://mui.com/components/data-grid/group-pivot/#custom-grouping-column) where developers learn how to completely override the grouping cell by a link to [row grouping](https://deploy-preview-3277--material-ui-x.netlify.app/components/data-grid/group-pivot/#grouping-columns) named "Custom grouping column" in which only the `headerName` prop is replaced

Reply:
Yeah you are probably right

Reply:
I added the demo back and pointed the anchor to he h3